src/styles/main.scss

```
@@ -0,0 +1,59 @@
+.container {
+  width: 80vmin;
+  height: 80vmin;
+  border: 120px dotted black;
+  border-radius: 50%;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 80px auto;
+}
+
+.circle {
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  background-color: orange;
+  position: absolute;
+  left: 335px;
```

looks like circle is not in the center of watch
```suggestion
  left: 50%;
  transform: translate(-50%)
```